server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -194,6 +194,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion DATA_STREAM_INDEX_VERSION_DEPRECATION_CHECK = def(8_788_00_0);
     public static final TransportVersion ADD_COMPATIBILITY_VERSIONS_TO_NODE_INFO = def(8_789_00_0);
     public static final TransportVersion VERTEX_AI_INPUT_TYPE_ADDED = def(8_790_00_0);
+    public static final TransportVersion FAST_REFRESH_ADAPT = def(8_791_00_0);
```

Comment:
nit: I would probably prefer to call this `FAST_REFRESH_RCO_2` just to signal that this is the successor of the prior transport version.

Reply:
Done.